src/gui/symbology/qgssymbollevelsdialog.h

```diff
@@ -33,7 +34,7 @@ class GUI_EXPORT QgsSymbolLevelsDialog : public QDialog, private Ui::QgsSymbolLe
     Q_OBJECT
   public:
     //! \note not available in Python bindings
-    QgsSymbolLevelsDialog( const QgsLegendSymbolList &list, bool usingSymbolLevels, QWidget *parent = nullptr ) SIP_SKIP;
+    QgsSymbolLevelsDialog( QgsFeatureRenderer *renderer, bool usingSymbolLevels, QWidget *parent = nullptr ) SIP_SKIP;
```

Comment:
Probably this does not need to have SIP_SKIP anymore - I think it was there for legacy reasons when the list was a complex type requiring conversion code.

Reply:
@wonder-sk , done. 

I've also moved the renderer->setUsingSymbolLevels() call to take place in the apply() function instead of QgsRendererWidget::showSymbolLevelsDialog(...), much cleaner that way.